examples/edge-functions/supabase/functions/telegram-bot/README.md

```diff
@@ -0,0 +1,13 @@
+# telegram-bot
+
+Try it out: https://t.me/supabase_example_bot
+
+![demo](https://obuldanrptloktxcffvn.supabase.co/storage/v1/object/public/images/supabase_community/telegram_bot/supa_telegram_bot.gif?t=2022-05-20T05:27:04.904Z)
```

Comment:
doesn't seem to be showing? also let's commit the .gif to the repo in case the project gets deleted